Week 1/Weekly Practice Problems (Week 1)/0709: To Lower Case/judah_silva.cpp

```diff
@@ -0,0 +1,11 @@
+class Solution {
+public:
+    string toLowerCase(string s) {
+        for (int i = 0; i < s.length(); i++) {
+            if (s[i] >= 65 && s[i] <= 90) {
+                s[i] += 32;
```

Comment:
+32 is a better operation here, but you can also do the addition this way:
```suggestion
                s[i] += 'a' - 'A';
```
Or 'A' - 'a'.